CMake/crosscompile-arch-config.cmake

```diff
@@ -63,14 +63,24 @@ elseif(ARCH STREQUAL "RPI3" OR ARCH STREQUAL "CORTEXA53")
   set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -mtune=cortex-a53 -ftree-vectorize")
   set(OPENBLAS_TARGET "CORTEXA53")
   set(OPENBLAS_BINARY "64")
-elseif(ARCH STREQUAL "RPI4" OR ARCH STREQUAL "CORTEXA72")
+elseif(ARCH STREQUAL "RPI4")
   set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -march=armv8.2-a+crypto+fp16+rcpc+dotprod -fasynchronous-unwind-tables")
   set(OPENBLAS_TARGET "CORTEXA72")
   set(OPENBLAS_BINARY "64")
+elseif(ARCH STREQUAL "CORTEXA72")
+  set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -mtune=cortex-a72 -ftree-vectorize")
+  set(OPENBLAS_TARGET "CORTEXA72")
+  set(OPENBLAS_BINARY "64")
 elseif(ARCH STREQUAL "JETSONAGX" OR ARCH STREQUAL "CORTEXA76")
   set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -mtune=cortex-a76 -ftree-vectorize")
   set(OPENBLAS_TARGET "CORTEXA76")
   set(OPENBLAS_BINARY "64")
+elseif(ARCH STREQUAL "JETSONORIN" OR ARCH STREQUAL "CORTEXA78")
```

Comment:
Technically, I would like to remove or deprecate the fact that the user can specify the board name, user should only specify `CORTEXA78`. This should be enough.

If there are specific flags per board, then we need to figure out a better way to indicate them

Reply:
I see, so it will basically be the same as for OpenBLAS where you just specify the name of the processor.

If a user has extra flags they want, they should just specify them with `CMAKE_CXX_FLAGS` I suppose.

I removed all the board names in 9ae861bd26; let me know if that's what you were thinking.  If not I can change it, just let me know.

Reply:
I mean, you are using the achitecture which is fine and perfect in most cases, but if the raspberry pi 4 has a specific flags, you can use instead the processor name which is: BCM2711

Reply:
Sounds good---let me know what you think of f22f98dc93; I re-added `BCM2711` as a choice.